ansible/roles/node/tasks/main.yml

```diff
@@ -2,3 +2,11 @@
 - name: Register NodeJS
   shell: 'curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -'
   changed_when: false
```

Comment:
You can drop this now so it gives indication of state

Reply:
Removed.